### ISSUE TITLE
compile error issued

### ISSUE THREAD

**Issue by Junhee12** [enesozi/object-detection#6]
Hi, 

I am trying to depoly my computer.

When I command "make", I encountered compile fail.

I search a lot of related the problem. 

I couldn't find the solution of it. 

Please help me ~

The compile results the following :

nvcc -gencode arch=compute_30,code=sm_30 -gencode arch=compute_35,code=sm_35 -gencode arch=compute_50,code=[sm_50,compute_50] -gencode arch=compute_52,code=[sm_52,compute_52] -gencode arch=compute_61,code=[sm_61,compute_61]  -Iinclude/ -I3rdparty/stb/include -DGPU -I/usr/local/cuda/include/ -DCUDNN --compiler-options "-Wall -Wfatal-errors -Wno-unused-result -Wno-unknown-pragmas -fPIC -ffp-contract=fast -mavx -mavx2 -msse3 -msse4.1 -msse4.2 -msse4a -Ofast -fopenmp -DGPU -DCUDNN -I/usr/local/cudnn/include -fPIC" -c ./src/maxpool_layer_kernels.cu -o obj/maxpool_layer_kernels.o
nvcc -gencode arch=compute_30,code=sm_30 -gencode arch=compute_35,code=sm_35 -gencode arch=compute_50,code=[sm_50,compute_50] -gencode arch=compute_52,code=[sm_52,compute_52] -gencode arch=compute_61,code=[sm_61,compute_61]  -Iinclude/ -I3rdparty/stb/include -DGPU -I/usr/local/cuda/include/ -DCUDNN --compiler-options "-Wall -Wfatal-errors -Wno-unused-result -Wno-unknown-pragmas -fPIC -ffp-contract=fast -mavx -mavx2 -msse3 -msse4.1 -msse4.2 -msse4a -Ofast -fopenmp -DGPU -DCUDNN -I/usr/local/cudnn/include -fPIC" -c ./src/network_kernels.cu -o obj/network_kernels.o
nvcc -gencode arch=compute_30,code=sm_30 -gencode arch=compute_35,code=sm_35 -gencode arch=compute_50,code=[sm_50,compute_50] -gencode arch=compute_52,code=[sm_52,compute_52] -gencode arch=compute_61,code=[sm_61,compute_61]  -Iinclude/ -I3rdparty/stb/include -DGPU -I/usr/local/cuda/include/ -DCUDNN --compiler-options "-Wall -Wfatal-errors -Wno-unused-result -Wno-unknown-pragmas -fPIC -ffp-contract=fast -mavx -mavx2 -msse3 -msse4.1 -msse4.2 -msse4a -Ofast -fopenmp -DGPU -DCUDNN -I/usr/local/cudnn/include -fPIC" -c ./src/avgpool_layer_kernels.cu -o obj/avgpool_layer_kernels.o
g++ -Iinclude/ -I3rdparty/stb/include -DGPU -I/usr/local/cuda/include/ -DCUDNN -Wall -Wfatal-errors -Wno-unused-result -Wno-unknown-pragmas -fPIC -ffp-contract=fast -mavx -mavx2 -msse3 -msse4.1 -msse4.2 -msse4a -Ofast -fopenmp -DGPU -DCUDNN -I/usr/local/cudnn/include -fPIC obj/http_stream.o obj/gemm.o obj/utils.o obj/dark_cuda.o obj/convolutional_layer.o obj/list.o obj/image.o obj/activations.o obj/im2col.o obj/col2im.o obj/blas.o obj/crop_layer.o obj/dropout_layer.o obj/maxpool_layer.o obj/softmax_layer.o obj/data.o obj/matrix.o obj/network.o obj/connected_layer.o obj/cost_layer.o obj/parser.o obj/option_list.o obj/darknet.o obj/detection_layer.o obj/captcha.o obj/route_layer.o obj/writing.o obj/box.o obj/nightmare.o obj/normalization_layer.o obj/avgpool_layer.o obj/coco.o obj/dice.o obj/yolo.o obj/detector.o obj/layer.o obj/compare.o obj/classifier.o obj/local_layer.o obj/swag.o obj/shortcut_layer.o obj/activation_layer.o obj/rnn_layer.o obj/gru_layer.o obj/rnn.o obj/rnn_vid.o obj/crnn_layer.o obj/demo.o obj/tag.o obj/cifar.o obj/go.o obj/batchnorm_layer.o obj/art.o obj/region_layer.o obj/reorg_layer.o obj/reorg_old_layer.o obj/super.o obj/voxel.o obj/tree.o obj/yolo_layer.o obj/upsample_layer.o obj/lstm_layer.o obj/image_opencv.o obj/convolutional_kernels.o obj/activation_kernels.o obj/im2col_kernels.o obj/col2im_kernels.o obj/blas_kernels.o obj/crop_layer_kernels.o obj/dropout_layer_kernels.o obj/maxpool_layer_kernels.o obj/network_kernels.o obj/avgpool_layer_kernels.o -o darknet -lm -pthread -lgomp -L/usr/local/cuda/lib64 -lcuda -lcudart -lcublas -lcurand -L/usr/local/cudnn/lib64 -lcudnn -lstdc++ 
obj/network.o: In function `resize_network':
network.c:(.text+0x1b87): undefined reference to `resize_conv_lstm_layer'
network.c:(.text+0x1d27): undefined reference to `resize_gaussian_yolo_layer'
network.c:(.text+0x1da7): undefined reference to `resize_scale_channels_layer'
network.c:(.text+0x1dbb): undefined reference to `resize_sam_layer'
obj/network.o: In function `num_detections':
network.c:(.text+0x2457): undefined reference to `gaussian_yolo_num_detections'
obj/network.o: In function `fill_network_boxes':
network.c:(.text+0x2d93): undefined reference to `get_gaussian_yolo_detections'
obj/network.o: In function `free_network_recurrent_state':
network.c:(.text+0x514e): undefined reference to `free_state_conv_lstm'
obj/network.o: In function `randomize_network_recurrent_state':
network.c:(.text+0x51fe): undefined reference to `randomize_state_conv_lstm'
obj/network.o: In function `remember_network_recurrent_state':
network.c:(.text+0x52ae): undefined reference to `remember_state_conv_lstm'
obj/network.o: In function `restore_network_recurrent_state':
network.c:(.text+0x532e): undefined reference to `restore_state_conv_lstm'
obj/parser.o: In function `parse_conv_lstm':
parser.c:(.text+0x12e2): undefined reference to `make_conv_lstm_layer'
obj/parser.o: In function `parse_gaussian_yolo':
parser.c:(.text+0x2a9b): undefined reference to `make_gaussian_yolo_layer'
obj/parser.o: In function `parse_scale_channels':
parser.c:(.text+0x4f14): undefined reference to `make_scale_channels_layer'
obj/parser.o: In function `parse_sam':
parser.c:(.text+0x503f): undefined reference to `make_sam_layer'
collect2: error: ld returned 1 exit status
Makefile:144: recipe for target 'darknet' failed
make: *** [darknet] Error 1
